tests/test_differentiation.py

```diff
@@ -65,6 +65,54 @@ def test_adjoint_diff(n_qubits: int, n_layers: int) -> None:
     # )
 
 
+@pytest.mark.parametrize("n_qubits", [3])
+@pytest.mark.parametrize("batch_size", [1])
+@pytest.mark.parametrize("ops_op", [pyq.Z, pyq.Y])
+@pytest.mark.parametrize("dtype", [torch.complex128])
```

Comment:
Ditto.

Reply:
I will add complex64 whenever we are sync with #220